planner/core/util.go

```diff
@@ -308,6 +308,19 @@ func extractStringFromUint64Slice(slice []uint64) string {
 	return strings.Join(l, ",")
 }
 
+// extractStringFromBoolSlice helps extract string info from bool slice.
+func extractStringFromBoolSlice(slice []bool) string {
+	if len(slice) < 1 {
+		return ""
+	}
+	l := make([]string, 0, len(slice))
+	for _, k := range slice {
+		l = append(l, fmt.Sprintf(`%t`, k))
+	}
+	sort.Strings(l)
```

Comment:
What is the purpose of sort?

Reply:
In unnormal case, user may query `is_leader in(0,1)`, sort can keep the order of result string.